Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1654 @@
+            get : function() {
+                return this._maximumMemoryUsage;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                Check.typeOf.number.greaterThanOrEquals('value', value, 0);
+                //>>includeEnd('debug');
+
+                this._maximumMemoryUsage = value;
+            }
+        },
+
+        /**
+         * The tileset's bounding volume.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {TileBoundingVolume}
+         * @readonly
+         *
+         * @exception {DeveloperError} The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.
+         */
+        boundingVolume : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
+                }
+                //>>includeEnd('debug');
+
+                return this._root._boundingVolume;
+            }
+        },
+
+        /**
+         * The tileset's bounding sphere.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {BoundingSphere}
+         * @readonly
+         *
+         * @exception {DeveloperError} The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.
+         */
+        boundingSphere : {
+            get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('The tileset is not loaded.  Use Cesium3DTileset.readyPromise or wait for Cesium3DTileset.ready to be true.');
+                }
+                //>>includeEnd('debug');
+
+                return this._root.boundingSphere;
+            }
+        },
+
+        /**
+         * A 4x4 transformation matrix that transforms the tileset's root tile.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Matrix4}
+         * @default Matrix4.IDENTITY
+         */
+        modelMatrix : {
+            get : function() {
+                return this._modelMatrix;
+            },
+            set : function(value) {
+                this._modelMatrix = Matrix4.clone(value, this._modelMatrix);
+                if (defined(this._root)) {
+                    // Update the root transform right away instead of waiting for the next update loop.
+                    // Useful, for example, when setting the modelMatrix and then having the camera view the tileset.
+                    this._root.updateTransform(this._modelMatrix);
+                }
+            }
+        },
+
+        /**
+         * Returns the time, in seconds, since the tileset was loaded and first updated.
```

Comment:
Yes, let's use ms in the spec as well.  In the spec, also add an implementation note about the precision.